src/device-registry/models/Airqloud.js

```diff
@@ -50,9 +50,20 @@ const metadataSchema = new Schema(
   { _id: false }
 );
 
+const pointSchema = new Schema(
```

Comment:
Updating the schema to cater for the new field (center_point) which will handle the center of an AirQloud